<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameFinishWidget</class>
 <widget class="QWidget" name="GameFinishWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="radar_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Radar" name="periceive_radar" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Radar" name="emotion_radar" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="curve_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Curve" name="periceive_curve" native="true"/>
     </item>
     <item>
      <widget class="Curve" name="emotion_curve" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Curve</class>
   <extends>QWidget</extends>
   <header>curve.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Radar</class>
   <extends>QWidget</extends>
   <header>radar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
